src/prop/minisat/sat_proof_manager.cpp

```diff
@@ -526,11 +524,11 @@ void SatProofManager::finalizeProof(Node inConflictNode,
       {
         continue;
       }
-      auto it = d_cnfStream->getTranslationCache().find(link.first);
-      if (it != d_cnfStream->getTranslationCache().end())
+      if (d_cnfStream->hasLiteral(link.first))
```

Comment:
This change potentially introduces unnecessary overhead (it adds an additional lookup to `d_nodeToLiteralMap` that is redundant).